content/docs/overview/_index.md

```diff
@@ -0,0 +1,75 @@
+
+The Autonity client extends the underlying Ethereum protocol at application, blockchain, and communication layers to implement:
+
+- A dual coin tokenomic model providing native protocol coin for gas payment and staking.
+- Delegated proof of stake consensus protocol (based on the Tendermint Byzantine fault tolerant consensus algorithm) with dynamic committee selection.
+- A liquid staking model: liquid newton receives rewards from staking and is transferrable, bringing liquidity and capital efficiency benefits to staking.
+- Protocol governance logic.
+
+For explanations of core concepts refer to:
+
+- [Architecture](/concepts/architecture/) for an Autonity system's technical architecture and protocol primitives.
+- [System model](/concepts/system-model/) for the functions of participant nodes and blockchain primitives comprising the structure of an Autonity system's peer-to-peer network and distributed ledger.
+- [Client](/concepts/client/) for an overview of Autonity Go Client features.
+- [Consensus](/architecture/consensus/) for the protocol's implementation of the Tendermint BFT consensus algorithm and consensus committee selection.
+- [Validator](/architecture/validator) for the role of validators and the functions they perform in securing an Autonity network.
+- [Staking](/architecture/staking/) for protocol staking mechanisms and Autonity's model for liquid staking.
+- [Protocol assets](/concepts/protocol-assets/) for the protocol's native coins and tokens:  _Auton_ utility coin, _Newton_ stake token, and _Liquid Newton_ liquid stake token.
+- The [Glossary](/glossary/#participant) for definitions of terms and concepts used in the documentation.
+
+## System actors
+
+The actors of an Autonity network are:
+
+- [Account Holders](/account-holders/): externally owned accounts of human users of the system. Account holders are the users interacting with an Autonity network: operating node and validation infrastructure, deploying and using decentralised applications, building and contributing to the community and ecosystem.
+
+- [Node operators](/node-operatorss/): network peer nodes running the AGC client software, forming the backbone of the networks p2p infrastructure and shared system ledger. Nodes store an up-to-date copy of the system state and may be run as public rpc endpoints providing open access to an Autonity network.
+
+- [Validators](/validators/): nodes with bonded stake forming the validation infrastructure proposing and maintaining system state. Active validator nodes are candidates for selection to the network's consensus committee. The consensus committee is responsible for proposing and deciding on new blocks.
+
+- [Staking](/staking/): stake delegators hold  _Newton_ stake token and delegate stake to one or more validators in order to secure the network. Stake holders receive _Liquid Newton_ for bonded stake.
```

Comment:
`Stakers` since we are listing actors?
```suggestion
- [Stakers](/staking/): stake delegators hold  _Newton_ stake token and delegate stake to one or more validators in order to secure the network. Stake holders receive _Liquid Newton_ for bonded stake.
```

Reply:
Yes.